src/fractionatedmorse.rs

```diff
@@ -0,0 +1,333 @@
+//! The Fractionated Morse cipher builds upon Morse code, a well-known method for encoding text which
+//! can then be sent across simple visual or audio channels.
+//! 
+//! The Fractionated Morse cipher does not produce a one-to-one mapping of plaintext characters to
+//! ciphertext characters and is therefore slightly more secure than a simple substitution cipher.
+//! In addition to this, it allows many non-alphabetic symbols to be encoded.
+use common::cipher::Cipher;
+use common::alphabet;
+use common::keygen::generate_keyed_alphabet;
+use std::ascii::AsciiExt;
+
+// The Morse alphabet.
+const MORSE_ALPHABET: [(char, &str); 50] = [

```

Comment:
Thinking about it, I would prefer if this was moved to its own file in common called `morse_alphabet` with similar access functions to the `alphabet` module.